documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -2720,6 +2706,16 @@ __OGM default configuration__ is transactional.
 We always recommend using transactional caches, at least unless you are confident that data consistency is not important for your use case.
 ====
 
+==== Remote Query Capability
+
+Hibernate OGM backend is able to run the queries it needs to materialize relations.
+From version __5.4.0.Alpha2__ the dialect is capable to translate JPQL or Criteria queries to the Infinispan remote queries.
+But there are still some limitations:
+
+* We don't support queries on polymorphic entities using TABLE_PER_CLASS inheritance strategy
+* Selecting from associations is not yet implemented
+* Query on Byte filed is not currently supported
```

Comment:
s/filed/fields/
s/is not currently supported/is currently unsupported/

Reply:
What do you mean when you say `selecting from association`?

Reply:
> What do you mean when you say `selecting from association`?

I understand now what you mean, I think a quick example will make it clearer. Something like:
Selecting from associations is not yet implemented (example: select h.author.name from Hypothesis h where author is a one-to-one association)